src/main/java/org/opensearch/knn/index/vectorvalues/KNNVectorValues.java

```diff
@@ -82,7 +82,6 @@ public int bytesPerVector() {
      * </pre>
      * @return long
      */
-    @Deprecated
     public long totalLiveDocs() {
```

Comment:
if we removing deprecated from here, we should also update the java docs.

Reply:
Missed as there was no @deprecated javadoc tag. The convention is to add @deprecated in javadocs tag  and give a reason next to it instead of combining it with existing